config/locales/zh.yml

```diff
@@ -1698,12 +1698,12 @@ two_factor_authentication.login_options.webauthn: 安全密钥
 two_factor_authentication.login_options.webauthn_info: 使用你的安全密钥来访问账户
 two_factor_authentication.login_options.webauthn_platform: 人脸或触摸解锁
 two_factor_authentication.login_options.webauthn_platform_info: 不用一次性代码，而是用你的面孔或指纹来访问你的账户。
-two_factor_authentication.max_backup_code_login_attempts_reached: 为了你的安全，你的账户暂时被锁住，因为你错误输入备用代码太多次。
-two_factor_authentication.max_generic_login_attempts_reached: 为了你的安全，你的账户暂时被锁住。
-two_factor_authentication.max_otp_login_attempts_reached: 为了你的安全，你的账户暂时被锁住，因为你错误输入一次性代码太多次。
-two_factor_authentication.max_otp_requests_reached: 为了你的安全，你的账户暂时被锁住，因为你要求一次性代码太多次。
-two_factor_authentication.max_personal_key_login_attempts_reached: 为了你的安全，你的账户暂时被锁住，因为你错误输入个人密钥太多次。
-two_factor_authentication.max_piv_cac_login_attempts_reached: 为了你的安全，你的账户暂时被锁住，因为你错误提供 piv/cac 凭据太多次。
+two_factor_authentication.max_backup_code_login_attempts_reached: 出于安全考虑，你的账户暂时被锁住，因为你错误输入备用代码太多次。
+two_factor_authentication.max_generic_login_attempts_reached: 出于安全考虑，你的账户暂时被锁住。
+two_factor_authentication.max_otp_login_attempts_reached: 出于安全考虑，你的账户暂时被锁住，因为你错误输入一次性代码太多次。
+two_factor_authentication.max_otp_requests_reached: 出于安全考虑，你的账户暂时被锁住，因为你要求一次性代码太多次。
+two_factor_authentication.max_personal_key_login_attempts_reached: 出于安全考虑，你的账户暂时被锁住，因为你错误输入个人密钥太多次。
```

Comment:
English: "For your security, your account is temporarily locked because you have entered the personal key incorrectly too many times."

Reply:
![Screenshot 2024-11-07 at 3 43 02 PM](https://github.com/user-attachments/assets/0ed68dfb-7f80-4fd7-9b81-ebd36046f4d4)